src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf

```diff
@@ -374,6 +374,14 @@
                 <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
                 <target>Liczba elementów w tym zbiorze powinna być wielokrotnością {{ compared_value }}.</target>
             </trans-unit>
+            <trans-unit id="97">
+                <source>This value should satisfy at least one of the following constraints:</source>
+                <target>Ta wartość powinna spełniać co najmniej jedną z następujących reguł:</target>
```

Comment:
@pizzaminded Since you've helped me before, maybe you could have a look at this translation? I had doubts whether to use `ograniczenie` instead of `reguła` (for translating `constraint`), but I think that the second one is more natural and commonly used.

Reply:
For me, ``reguła`` sounds better.   ``Ograniczenie``  is more about limiting something (e.g  count of choices etc.) than "fitting" to requirements.